python/py-colorama/Portfile

```diff
@@ -4,7 +4,7 @@ PortSystem          1.0
 PortGroup           python 1.0
 
 name                py-colorama
-version             0.3.7
+version             0.3.9
```

Comment:
The latest version is 0.4.1 [1]. Any reason not using that?

[1] https://pypi.org/project/colorama/0.4.1/

Reply:
unfortunately, py-tmuxp (which i'll push later) requires colorama 0.3.9 :(

Reply:
Here are some options:

* Backport https://github.com/tmux-python/tmuxp/commit/eab92566c25b1a430553b95960367dd82b5ba69e in the upcoming tmuxp port.
* Keep it as-is, using colorama 0.3.9. It's better to add a comment here to avoid accidental updates in the future.

I prefer the first option, and I'm not against the second.

Reply:
thanks for the hint. i'll try the first option.

Reply:
the first version worked :) thanks
tmuxp is up and running on MacOS
finally i can get rid of tmuxinator :)
i'll wait until it merges and then push py-tmuxp

thanks again